### PR TITLE
[docs] add a note on dangling volumes after changing era

### PR DESCRIPTION
**Pull request by rustielin**
### Description

This is relevant to https://github.com/aptos-labs/aptos-core/issues/5508. While it's not recommended/common for node operators to be wiping their storage, this doc change clarifies a quirk with our k8s setup.

If operators wipe their storage on k8s changing the `era`, they must also check if there are any dangling volumes from previous eras and delete them accordingly. Otherwise, their costs might stack up.

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-aws.md [hunk 1]
```diff
@@ -5,7 +5,15 @@ slug: "run-validator-node-using-aws"
 
 # On AWS
 
-This is a step-by-step guide to install an Aptos node on AWS. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+This is a step-by-step guide to install an Aptos node on Amazon Web Services (AWS). Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+
+:::caution Did you set up your AWS account?
```
Comment (clay-aptos) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-aws.md:10]: This seems redundant with the prerequisites below. Are folks actually forgetting to create accounts?

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md [hunk 2]
```diff
@@ -5,16 +5,21 @@ slug: "run-validator-node-using-azure"
 
 # On Azure
 
-This is a step-by-step guide to install an Aptos node on Azure. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+This is a step-by-step guide to install an Aptos node on Microsoft Azure. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
 
-:::danger Did you set up your Azure account?
+:::caution Did you set up your Azure account?
```
Comment (clay-aptos) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md:10]: Same comment as above - seems superfluous.

File: developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md [hunk 3]
```diff
@@ -5,16 +5,21 @@ slug: "run-validator-node-using-azure"
...
 This guide assumes that you already have Azure account setup.
 :::
 
+:::danger Do you have stale volumes after bumping your deployment's era?
```
Comment (clay-aptos) [inline developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md:14]: This is also redundant now that I have added an account to the Before list for consistency with the other cloud setup docs.

**Review by clay-aptos — APPROVED** [review]
Thanks so much, Rustie! This LGM with some changes I made directly. I also included some optional comments on removing the now redundant account warnings.

I wonder if we should add a similar warning regarding dangling volumes to:
https://aptos.dev/nodes/validator-node/operator/update-validator-node

Finally, I think we should also announce this change in:
https://aptos.dev/whats-new-in-docs/

I can take care of the last two tasks there. Please let me know what you think. Thanks!

**Comment by rustielin** [comment]
@clay-aptos thanks for the review! 

Re copying to the update validators doc, I don't think this is necessary since the issue only arises when the 'era' variable is toggled, which is only referenced in these docs iiuc.

**Pull request merged**